Utilities/HttpServer.cs

```diff
@@ -113,6 +168,38 @@ public class HttpServer {
 
       HttpListenerRequest request = context.Request;
 
+      if (request.HttpMethod == "POST") {

```

Comment:
Going to just put a comment for the entire block here, but can you add some comments to this block so we know what is going on? I get the general idea here but some example output would be welcome.

Reply:
Are you ok with the commit message, or should I add comments to the code?

Reply:
Comments in the code is better as it's not that easy to read back the comments from the commit. I think some general comments regarding the input and output should be the minimum 👍

Reply:
Looking at the previous code, there are basically no comments (so I would like to stay with that style). And I'd rather have the code speak for itself in regards to its functionality. If you think there is something unclear I'd rather change the code to make it more obvious what is happening ( which also was the reason to split the block into multiple functions )

Reply:
I've added comments into the code now.